scripts/dpss_delay_filter_run.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```

Comment:
That is one option. I am a bit concerned with condensing the scripts from a usability perspective since I think different sets of parameters are best for different filtering modes. For example, linear filters tend to have issues with flagged edges so they work best when skip_flagged_edges is True. I'm worried that non-experts won't know to set these defaults and run into problems.

Reply:
I've decided to go with a system with a single script that sets different defaults based on the filtering mode.